ipn/message.go

```diff
@@ -241,7 +241,7 @@ func (bc *BackendClient) GotNotifyMsg(b []byte) {
 		log.Fatalf("BackendClient.Notify: cannot decode message (length=%d, %#q): %v", len(b), b, err)
 	}
 	if n.Version != version.Long && !bc.AllowVersionSkew {
-		vs := fmt.Sprintf("GotNotify: Version mismatch! frontend=%#v backend=%#v",
+		vs := fmt.Sprintf("GotNotify: Version mismatch! Please try to reboot if you just updated Tailscale. frontend=%#v backend=%#v",
```

Comment:
Package install is expected to stop the running tailscaled and start the new version immediately.

We do not require a reboot to update the tailscale package, and don't want to print a message telling people to do so.